forge/licensing/loader.js

```diff
@@ -6,6 +6,11 @@ const jwt = require('jsonwebtoken')
 
 const LICENSE_ISSUER = 'FlowForge Inc.'
 
+const MQTT_CLIENT_DEFAULT_ALLOWANCE = {
+    teams: 5,
```

Comment:
As don't have teams tier licensing anymore, is this just to cover legacy license?

Reply:
Yeah - given 'teams' has been a historically valid license type, just covering our bases.